public/config.js

```diff
@@ -3,15 +3,15 @@
 // eslint-disable-next-line no-unused-vars
 var jujuDashboardConfig = {
   // API host to allow app to connect and retrieve models
-  baseControllerURL: "jimm.jujucharms.com",
+  baseControllerURL: "localhost:17070",
```

Comment:
Oops :) Can you remove these changes plz.

Reply:
Done